Integrations/Perch/Perch.yml

```diff
@@ -0,0 +1,415 @@
+      description: Communities to report the indicator to.
+      isArray: false
+      name: communities
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Confidence of findings.
+      isArray: false
+      name: confidence
+      predefined:
+      - LOW
+      - MEDIUM
+      - HIGH
+      required: true
+      secret: false
+    - auto: PREDEFINED
+      default: false
+      description: Type of indicator.
+      isArray: false
+      name: type
+      predefined:
+      - IP
+      - Domain
+      - URL
+      - REGEX
+      - File Hash
+      required: true
+      secret: false
+    - default: false
+      description: Indicator's value.
+      isArray: false
+      name: value
+      required: true
+      secret: false
+    - default: false
+      description: Title for the indicator.
+      isArray: false
+      name: title
+      required: true
+      secret: false
+    - default: false
+      description: Description of the indicator.
+      isArray: false
+      name: description
+      required: true
+      secret: false
+    - default: false
+      description: TLP of the Indicator.
+      isArray: false
+      name: tlp
+      required: true
+      secret: false
+    - default: false
+      description: Operator of the indicator.
+      isArray: false
+      name: operator
+      required: false
+      secret: false
+    - default: false
+      description: When the indicator was first sighted.
+      isArray: false
+      name: first_sighting
+      required: false
+      secret: false
+    - default: false
+      description: Send a summary of the indicator via email.
+      isArray: false
+      name: email_summary
+      required: false
+      secret: false
+    deprecated: false
+    description: Creates an indicator in Perch.
+    execution: false
+    name: perch-create-indicator
+    outputs:
+    - contextPath: Perch.Indicator.Confidence
+      description: Confidence of indicator.
+      type: Unknown
+    - contextPath: Perch.Indicator.UpdatedAt
+      description: Time Indicator was last updated.
+      type: Date
+    - contextPath: Perch.Indicator.TLP
+      description: TLP of the Indicator.
+      type: String
+    - contextPath: Perch.Indicator.Title
+      description: Title of the indicator.
+      type: String
+    - contextPath: Perch.Indicator.ID
+      description: ID of the indicator.
+      type: Number
+    - contextPath: Perch.Indicator.CreatedAt
+      description: Date indicator was created.
+      type: Date
+    - contextPath: Perch.Indicator.Team
+      description: ID of team.
+      type: Number
+    - contextPath: Perch.Indicator.PerchID
+      description: ID given by perch of the incident.
+      type: String
+    - contextPath: Perch.Indicator.CreatedBy
+      description: ID of incident's creator.
+      type: Number
+  dockerimage: demisto/python3:3.7.2.214
+  isfetch: true
+  runonce: false
+  script: ''
+  type: python
```

Comment:
please add python3 subtype  see https://github.com/demisto/content/blob/9a08201d3246ec6cef0034472e10160d3a9f6041/Scripts/CherwellIncidentUnlinkTask/CherwellIncidentUnlinkTask.yml
for example